package.json

```diff
@@ -17,61 +17,61 @@
     "reselect": "4.0.0"
   },
   "devDependencies": {
-    "@babel/core": "7.4.5",
-    "@babel/plugin-proposal-async-generator-functions": "7.2.0",
-    "@babel/preset-react": "7.0.0",
-    "acorn": "6.1.1",
+    "@babel/core": "7.7.2",
+    "@babel/plugin-proposal-async-generator-functions": "7.7.0",
+    "@babel/preset-react": "7.7.0",
+    "acorn": "7.1.0",
     "babel-eslint": "10.0.3",
     "babel-loader": "8.0.6",
     "babel-plugin-jsm-to-commonjs": "0.5.0",
     "babel-plugin-jsm-to-esmodules": "0.6.0",
     "chai": "4.2.0",
     "chai-json-schema": "1.5.1",
     "cpx": "1.5.0",
-    "enzyme": "3.9.0",
-    "enzyme-adapter-react-16": "1.13.2",
-    "eslint": "6.2.2",
-    "eslint-config-prettier": "4.2.0",
+    "enzyme": "3.10.0",
+    "enzyme-adapter-react-16": "1.15.1",
+    "eslint": "6.6.0",
+    "eslint-config-prettier": "6.7.0",
```

Comment:
`Prettier` is safe to update but we should keep the configs in sync with [m-c](https://searchfox.org/mozilla-central/source/package.json).